sdk/cosmos/azure-cosmos-spark_3-4_2-12/README.md

```diff
@@ -0,0 +1,172 @@
+# Azure Cosmos DB OLTP Spark 3 connector
+
+## Azure Cosmos DB OLTP Spark 3 connector for Spark 3.3
+**Azure Cosmos DB OLTP Spark connector** provides Apache Spark support for Azure Cosmos DB using 
+the [SQL API][sql_api_query].
+[Azure Cosmos DB][cosmos_introduction] is a globally-distributed database service which allows 
+developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
+
+If you have any feedback or ideas on how to improve your experience please let us know here:
+https://github.com/Azure/azure-sdk-for-java/issues/new
+
+### Documentation
+
+- [Getting started](https://aka.ms/azure-cosmos-spark-3-quickstart)
+- [Catalog API](https://aka.ms/azure-cosmos-spark-3-catalog-api)
+- [Configuration Parameter Reference](https://aka.ms/azure-cosmos-spark-3-config)
+
+[//]: # (//TODO: add more sections)
+[//]: # (//TODO: Enable Client Logging)
+[//]: # (//TODO: Examples)
+[//]: # (//TODO: Next steps)
+[//]: # (//TODO: Key concepts)
+[//]: # (//TODO: Azure Cosmos DB Partition)
+[//]: # (//TODO: Troubleshooting)
+
+### Version Compatibility
+
+#### azure-cosmos-spark_3-4_2-12
+| Connector | Supported Spark Versions | Minimum Java Version | Supported Scala Versions | Supported Databricks Runtimes |
+|-----------|--------------------------| -------------------- | -----------------------  |-------------------------------|
+| 4.20.0    | 3.4.0                    | 8                    | 2.12                     | 11.\*                         |
+
+#### azure-cosmos-spark_3-3_2-12
+| Connector | Supported Spark Versions | Minimum Java Version | Supported Scala Versions | Supported Databricks Runtimes |
+|-----------|--------------------------| -------------------- | -----------------------  |-------------------------------|
+| 4.19.0    | 3.3.0                    | 8                    | 2.12                     | 11.\*                         |
```

Comment:
4.19 has not been shipped for just Spark 3.3 - either it exisst across Spark 3.1, 3.2 and 3.3 or not yet at all